### PR TITLE
[Snyk + GH Actions] Update requirements

### PR DESCRIPTION
**Pull request by btylerburton**
Update requirements
- Updated `requirements.in` + `requirements.txt`
- Auto-generated by [snyk.yml][1]

[1]: https://github.com/gsa/catalog.data.gov/.github/workflows/snyk.yml

### CODE REVIEW

File: ckan/requirements.txt [hunk 1]
```diff
@@ -4,18 +4,18 @@ Babel==2.10.3
...
+boto3==1.34.128
+botocore==1.34.128
 certifi==2024.6.2
 cffi==1.16.0
 chardet==5.2.0
 charset-normalizer==3.3.2
 ckan @ git+https://github.com/GSA/ckan.git@7159a872ba740069b768fcd2a43cde81a57ee492
 -e git+https://github.com/ckan/ckanext-archiver.git@cbfadf9fbf10405958fdef9f77a7faedc05aa20b#egg=ckanext_archiver
 ckanext-datagovcatalog==0.1.0
-ckanext-datagovtheme==0.2.22
```
Comment (jbrown-xentity) [inline ckan/requirements.txt:16]: I think we want this to be 24: https://pypi.org/project/ckanext-datagovtheme/0.2.24/

**Review by jbrown-xentity — APPROVED** [review]
We could move forward, but I would suggest this upgrade.

File: ckan/requirements.txt (hunk 1 above)
Comment (btylerburton) [reply ckan/requirements.txt:16]: I'm going to bump it in the other PR

**Comment by FuhuXia** [comment]
I was going to say we don't have to pin NumPy to 1.26.4, since it was the latest, but then [NumPy 2.0 was released two days ago](https://numpy.org/news)

**Pull request merged**